sponsorblock.lua

```diff
@@ -231,6 +245,7 @@ function getranges(_, exists, db, more)
     else
         sponsors = utils.subprocess({args = args})
     end
+    mp.msg.debug("Got: " .. dump(sponsors))
```

Comment:
We don't need this whole dump function when `sponsors.stdout` is the only thing we actually care about, already as a string.